code/web/src/modules/user/StyleSurvey.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Helmet } from "react-helmet";
+
+// UI Imports
+import { Grid, GridCell } from "../../ui/grid";
+import { H3, H4 } from "../../ui/typography";
+import Button from "../../ui/button";
+import Icon from "../../ui/icon";
+import { textLevel1 } from "../../ui/common/shadows";
+import { white, grey, grey2, grey3 } from "../../ui/common/colors";
+
+import { APP_URL } from "../../setup/config/env";
+import userRoutes from "../../setup/routes/user";
+
+
+class StyleSurvey extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: {
+        classy: 0,
+        artsy: 0,
+        punk: 0,
+        sporty: 0,
+        natureesque: 0
```

Comment:
Great choices for style options!

Reply:
December and I will add counter logic as the user select the choices of picture in the survey